SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+The Nuts node is currently under development. No versions are build, just use the master branch.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you discover a vulnerability, we would like to know about it so we can take steps to address it as quickly as possible.
+
+
+Note that the Nuts foundation does not host any instances of the nuts software. The foundation will facilitate in the communication
+with the Nuts software community in case of a software problem. We will try to connect with the node administrator in case of a configuration problem.
+
+Please do the following:
+
+  * E-mail your findings to responsible-disclosure@nuts.nl.
```

Comment:
maybe drop the link or make it an actual markdown link, now people might copy the postfixing dot by mistake?